Vic2ToHoI4/Data_Files/Configurables/converterFocuses.txt

```diff
@@ -0,0 +1,3309 @@
+shared_focus = {
```

Comment:
This file is upload to a capital C Configurables folder instead of the existing lower case one. This has confused github and will cause more problems in the future. Please rename the folder.

Reply:
Moved the files accordingly